server/model/monitor.js

```diff
@@ -348,25 +348,30 @@ class Monitor extends BeanModel {
                     bean.msg = dnsMessage;
                     bean.status = UP;
                 } else if (this.type === "push") {      // Type: Push
-                    const time = R.isoDateTime(dayjs.utc().subtract(this.interval, "second"));
-
-                    let heartbeatCount = await R.count("heartbeat", " monitor_id = ? AND time > ? ", [
-                        this.id,
-                        time
-                    ]);
-
-                    log.debug("monitor", "heartbeatCount" + heartbeatCount + " " + time);
-
-                    if (heartbeatCount <= 0) {
-                        // Fix #922, since previous heartbeat could be inserted by api, it should get from database
-                        previousBeat = await Monitor.getPreviousHeartbeat(this.id);
-
-                        throw new Error("No heartbeat in the time window");
-                    } else {
-                        // No need to insert successful heartbeat for push type, so end here
-                        retries = 0;
-                        this.heartbeatInterval = setTimeout(beat, beatInterval * 1000);
-                        return;
+                    log.debug(`[${this.name}] Checking monitor at ${dayjs().format("YYYY-MM-DD HH:mm:ss.SSS")}`);
+                    const bufferTime = 1000; // 1s buffer to accommodate clock differences
+                    // Fix #922, since previous heartbeat could be inserted by api, it should get from database
+                    previousBeat = await Monitor.getPreviousHeartbeat(this.id);
+                    //If the previous beat was nonexistent, down or pending we use the regular
```

Comment:
Be consistent with formatting
```suggestion
                    // If the previous beat was nonexistent, down or pending we use the regular
```